src/WebApi/WebApi/appsettings.json

```diff
@@ -14,5 +14,12 @@
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "Authentication": {
+    "Google": {
+      "ClientID": "172955704257-ug7p7mbjrd7ogkegvjfldnofal57q11b.apps.googleusercontent.com",
+      "ClientSecret": "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"
+    }
```

Comment:
I don't think this should be here 🙈 we can fill value in the pipeline if needed.
⚠️ ⚠️ ⚠️  if its really a secret please remove it also from a git history with an interactive rebase ⚠️ ⚠️ ⚠️